src/ape/cli/choices.py

```diff
@@ -29,6 +29,7 @@ class Alias(click.Choice):
 
     # noinspection PyMissingConstructor
```

Comment:
```suggestion
```

We can delete the Pycharm directives for now, usually `type: ignore` also appeases pycharm and we can put those back as needed